debian/rules

```diff
@@ -7,19 +7,24 @@ export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
 	dh $@ --with=gir,python3
 
 override_dh_autoreconf:
-	dh_autoreconf --as-needed
+	dh_autoreconf \
+		--as-needed \
+		debian/autogen.sh \
+		-- \
+		$(NULL)
 
 override_dh_auto_configure:
 	dh_auto_configure -- \
 		--disable-silent-rules \
 		--enable-docbook-docs \
 		--enable-gtk-doc \
 		--enable-installed-tests \
+		--enable-p2p \
 		--libexecdir=/usr/lib/flatpak \
 		--with-priv-mode=none \
 		--with-html-dir=/usr/share/doc/libflatpak-doc \
 		--with-privileged-group=sudo \
-		--with-system-bubblewrap=/usr/bin/bwrap \
+		--with-system-bubblewrap=bwrap \
```

Comment:
Another downstream change by us - https://github.com/endlessm/flatpak/commit/b197ff9b40d94c6a07dc19d168566344c78f682a. I'm not sure if this is still valid or not.

Reply:
This was fixed in the upstream commit 983b2a5db

Reply:
Ah, OK.